src/workloads/query/InWithVariedArraySize.yml

```diff
@@ -132,36 +130,28 @@ Actors:
           OperationCommand:
             createIndexes: *Collection
             indexes:
-            - key: {int: 1}
-              name: int_1
             - key: {randomDouble: 1}
```

Comment:
To have explode for sort, you also need to start the index with _id.

Reply:
Done.